src/aiida/cmdline/commands/cmd_code.py

```diff
@@ -234,34 +236,35 @@ def show(code):
 
 @verdi_code.command()
 @arguments.CODE()
-@arguments.OUTPUT_FILE(type=click.Path(exists=False))
-@click.option(
-    '--sort/--no-sort',
-    is_flag=True,
-    default=True,
-    help='Sort the keys of the output YAML.',
-    show_default=True,
-)
+@arguments.OUTPUT_FILE(type=click.Path(exists=False, path_type=pathlib.Path), required=False)
+@options.OVERWRITE()
```

Comment:
This might be a bit of hassle and maybe unnecessarily. This is what happens to user: 
1) Hit the error `FileExistsError` first time, 
2) discover there is such option as overwrite
3) run again with `--overwrite`

Instead one could easily just index files:
For example if file `mycomputer.yaml` exists just produce `mycomputer_1.yaml` if that exists just do `mycomputer_2.yaml`

Reply:
btw, this is what my browser does when I download files. They used to raise file exist in the past, and aske user for a new file name :)

Reply:
I am personally not a fan of this to be honest. If the user explicitly specified the output file, I wouldn't want the command to silently change it. Of course you could get around the "silently" a bit by printing a warning that the message was changed, but this will not help in scripting use cases.

Reply:
Thanks for the comments, @khsrali! There's no such thing as an annoying review :)

I see where you're coming from, but I'm not sure that I like the idea of automatically indexing the files, as that leads to the output name of the file being different from the label of the AiiDA entity, and I'd like those to be consistent. Suppose I run `verdi computer list` and I'm getting the output `mycomputer`. When I export it, I expect the file to be named `mycomputer.yaml`. Instead, if the file is called `mycomputer_1.yaml`, I might get confused (ofc we can inform the user, but still). Then, the user might still not be aware of the `--overwrite` flag, and then has to either run the command again using that, or `mv` his file to overwrite the original one or get the directory polluted with a bunch of files.

In the end, with all these commands that write to the disk, we have to make some decisions, e.g. file names, how to handle overwriting, etc., and there's no right or wrong, and people have different preferences. I don't have a strong opinion, but slightly prefer the current behavior. Let's see what @sphuber thinks.

Reply:
I see your points, my concern was mainly when the user is not specifying an `output_name` so he doesn't really care... But no strong opinion.